### DISCUSSION TITLE
Use model / relationship for status

### DISCUSSION THREAD

**Discussion by binaryfire** [mokhosh/filament-kanban#64] [Ideas]
Hi @mokhosh

I already mentioned this in the Filament Discord, but I figured I'd open a discussion so you can tag it as v3 :)

It'd be great if we can use a model / relationship for statuses instead of an enum. I have a multi-tenant app where each tenant needs to create their own statuses. This is common for project management tools.

**Comment by mokhosh** [comment]
Thanks. 
I definitely have this on the list.

**Reply by binaryfire** [reply]
@mokhosh Nice! It'd be good if status are sortable and users can re-order them by dragging the column titles. I'll probably have a status resource with a reorderable table view as well.

**Comment by sawirricardo** [comment]
Any ETA when? @mokhosh
